docs/using_unidist/unidist_on_mpi.rst

```diff
@@ -110,13 +110,32 @@ When initializing unidist this execution model gets transformed to Controller/Wo
 Unidist on MPI cluster
 """"""""""""""""""""""
 
-In order to run unidist on MPI in a cluster, there are two options.
+Regardless of the chosen usage model (SPMD model or Controller/Worker model),
+there are two options for running on a cluster.
 
-Controller/Worker model
------------------------
+Running with ``mpiexec`` command
+--------------------------------
+
+This option is the most preferred and customizable.
+
+Running is almost the same as in a single node, but, in addition,
+you should use the appropriate parameter for ``mpiexec``.
+This parameter differs depending on the MPI implementation used.
+
+* For Intel MPI or MPICH: ``-hosts host1,host2``. You can also see 
+`Controlling Process Placement with the Intel® MPI Library <https://www.intel.com/content/www/us/en/developer/articles/technical/controlling-process-placement-with-the-intel-mpi-library.html>`_ 
+or `MPICH wiki <https://github.com/pmodels/mpich/blob/main/doc/wiki/how_to/Using_the_Hydra_Process_Manager.md>`_ for deeper customization.
+
+* For OpenMPI: ``-host host1:n1,...,hostM:nM``
+where n1, ..., nM is the number of processes on each node, including system processes.
```

Comment:
Which system processes do you mean?

Reply:
Sorry, I meant Unidist service processes  (root and one or some monitors). It has fixed.